File: apps/server/check_db.js
const { createClient } = require('@supabase/supabase-js');

require('dotenv').config();
const url = process.env.SUPABASE_URL || 'https://vlpaszzbebgrfppklqml.supabase.co';
const key = process.env.SUPABASE_KEY;
if (!key) {
  console.error("ERROR: SUPABASE_KEY environment variable is not defined.");
}
const supabase = createClient(url, key || '');

async function check() {
  const { data, error } = await supabase.from('profiles').select('username, password, role');
  if (error) {
    console.error('ERROR:', error);
  } else {
    console.log('PROFILES:', JSON.stringify(data, null, 2));
  }
}

check();

File: apps/server/check_db.py
import os
from dotenv import load_dotenv
from supabase import create_client, Client

load_dotenv("d:/project_WKNsite/apps/server/.env")

url = os.environ.get("SUPABASE_URL")
key = os.environ.get("SUPABASE_KEY")

supabase: Client = create_client(url, key)

try:
    res = supabase.table("employees").select("fcm_token").limit(1).execute()
    print("Column exists:", res.data)
except Exception as e:
    print("Error:", e)
